src/DiffOp/Diff/Diff.php

```diff
@@ -12,6 +12,9 @@
  * Base class for diffs. Diffs are collections of DiffOp objects,
  * and are themselves DiffOp objects as well.
  *
+ * @SuppressWarnings(PHPMD.TooManyMethods)
```

Comment:
A limit of "under 10" methods per class? I find that extreme and not helpful. Can we please raise this?